test/test_jit.py

```diff
@@ -7680,6 +7680,37 @@ def test_no_type():
                 # type: () -> int
                 return torch.jit._unwrap_optional(None)
 
+    def test_list_of_optionals(self):
+        @torch.jit.script
```

Comment:
this test isn't really necessary right?

Reply:
Technically it's already being tested through torch/nn, but I think it's good to have modular tests

Reply:
Can you make these two tests to test_optional_conversion to keep a single place for testing those functionalities?